File: Core/Entities/DoctorHonAppr.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Core.Entities
{
    public class DoctorHonAppr : BaseEntity
    {
        public int InvestmentInitId { get; set; }
        [ForeignKey("InvestmentInitId")]
        public InvestmentInit InvestmentInit { get; set; }
        public int DoctorId { get; set; }
        [ForeignKey("DoctorId")]
        public DoctorInfo DoctorInfo { get; set; }
        public long HonAmount { get; set; }
        public string HonMonth { get; set; }
        public string Status { get; set; }
    }
}

File: client/src/app/shared/models/doctorHonAppr.ts

export interface IDoctorHonAppr {
    id: number;
    investmentInitId: number;
    doctorID: number;
    honMonth: string; 
    status: string; 
    honAmount: string; 
    setOn: Date;
}
 
export class DoctorHonAppr implements IDoctorHonAppr {
    id: number=0;
    investmentInitId: number;
    doctorID: number;
    honMonth: string; 
    status: string=null; 
    honAmount: string; 
    setOn: Date;
}